patches/net/minecraft/world/entity/ai/village/poi/PoiTypes.java.patch

```diff
@@ -29,7 +29,7 @@
  
```

Comment:
`+   protected static final Set<BlockState> f_218067_ = TYPE_BY_STATE.keySet();` in this same file looks suspicious.